common/js/src/logging/debug-dump.js

```diff
@@ -179,27 +179,28 @@ function dumpObject(value) {
  * @return {string}
  */
 GSC.DebugDump.dump = function(value) {
-  if (!goog.isDef(value))
```

Comment:
I think the problem is that GSC.PopupWindow.Server.createWindow makes the debug dump of all passed arguments, which really is a bad thing once we start passing the complex arguments (like pointing to DOM elements, which contain circular references) to it.
So just removing these debug dumps from createWindow should be enough to fix the issue.